config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+ 
+  get '/ingredients', to: 'ingredients#index', as: 'ingredients'
+  get '/recipes/:id', to: 'recipes#show', as: 'recipe'
```

Comment:
Good restful routes.

You do not need to add the aliases to your routes, especially since we are not allowing route helpers right now. Wait until the advanced routing class to determine when it is necessary.